addons/sale_crm/views/sale_order_views.xml

```diff
@@ -9,7 +9,7 @@
         <field name="context">{'search_default_opportunity_id': active_id, 'default_opportunity_id': active_id}</field>
     </record>
 
-    <record id="sale_view_inherit123" model="ir.ui.view">
+    <record id="view_order_form" model="ir.ui.view">
```

Comment:
If we really want to comply to guidelines, it should be sale_orer_view_form even if base view is called differently :) and name should be "sale.order.view.form.inherit.sale" . If we change it, I suggest to go full guidelines :D

Reply:
@tde-banana-odoo updated :)